src/components/pages/ProfilePage/components/ModalEditProfile/ModalEditProfile.tsx

```diff
@@ -0,0 +1,194 @@
+import { Dispatch, SetStateAction, useState } from 'react';
+import { Formik, Form, Field } from 'formik';
+
+import { Input, ModalBox, Dropdown } from 'components/common';
+import { Locations, PhotoUploader } from 'components/pages/Questionnaire';
+
+import { monthsForLocale } from '../../../../../utils/date';
+import { profileSchema } from './validate.schema';
+
+import SimpleCloseIcon from '../../../../../assets/ForCommonModal/closeButton.svg';
+import styles from './ModalEditProfile.module.css';
+
+interface Values {
+  name: string;
+  day: string;
+  year: string;
+  location: string;
+  about: string;
+}
+
+type SetFieldFunction = (field: string, value: string) => void;
+
+const mockValues = {
+  name: 'Nike Borzov',
+  day: '15',
+  month: '7',
+  year: '1980',
+  location: 'San Francisco, Argentina',
+  about: 'I am little horse',
+};
+
+const MyInput = ({ field, form, ...props }) => {
```

Comment:
наверное не удалила или я ошибаюсь
```suggestion
const MyInput = ({ field,  ...props }) => {
```

Reply:
Если я не вытяну form, то в props уйдет form, а мне надо только остаток

Reply:
ну окей